app/services/katello/pulp/repository.rb

```diff
@@ -238,6 +238,18 @@ def copy_units(destination_repo, units, options = {})
         override_config[:resolve_dependencies] = true if options[:resolve_dependencies]
 
         smart_proxy.pulp_api.extensions.send(content_type).copy(repo.pulp_id, destination_repo.pulp_id, ids: unit_ids, override_config: override_config)
+        if content_type == ::Katello::Erratum::CONTENT_TYPE
+          # copy the modules belonging to this errarta over (pulp 2.19 should handle this case out of the box)
+          # remove this code after 2.19
+          module_stream_ids = units.map do |erratum|
+            erratum.module_stream_objects.map(&:pulp_id)
+          end
+          module_stream_ids.flatten!
+          smart_proxy.pulp_api.extensions.module.copy(repo.pulp_id,
+                                                      destination_repo.pulp_id,
+                                                      ids: module_stream_ids,
+                                                      override_config: override_config)
+        end
```

Comment:
could you put this in the yum service class?  Maybe override this method and then call super() followed by this code?

Reply:
you sure thats a better approach ? Considering this is a stopgap

Reply:
I think so and then we can just delete the overridden method?

Reply:
moreover I am using same override config as errata

Reply:
ah, i guess its fine then, since this is temporary